go.mod

```diff
@@ -236,3 +236,5 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.14.3-0.20230601165947-6ce0bf390ce3 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+replace github.com/vishvananda/netlink => github.com/vishvananda/netlink v0.0.0-20240425164735-856e190dd707
```

Comment:
why doesn't it set version in `require` directly?

@hongliangl have you figured out whether the new version really affects the e2e test?

Reply:
Not yet, still testing it on my forked Antrea repo.

Reply:
I tried to change the version in `require` directly, but it is changed back to `v1.2.1-beta.2` after Goland resynced the dependency.

Reply:
Hongliang's PR is merged. No need to bump up netlink now.